README.md

```diff
@@ -136,6 +136,7 @@
 - [Bartender](http://www.macbartender.com/) - Organize your menu bar apps.
 - [BetterZip](http://macitbetter.com/) - A very capable and full-featured archive manager.
 - [DaisyDisk](http://www.daisydiskapp.com/) - Analyze disk usage and free up disk space.
+- [EtreCheck](http://www.etresoft.com/etrecheck) - Output system information and configuration to get more informed help from Apple support professionals. [![Open-Source Software][OSS Icon]](https://github.com/etresoft/EtreCheck)
```

Comment:
Nope. EtreCheck is a run-once system information dump whereas iStat Menus is a system monitor that runs constantly. Two completely different things. ;) Think about it this way: you request help from someone online (StackExchange, Apple Support Community, ...) and get asked about your setup. You run EtreCheck and get a dump of all relevant information an Apple Professional will need.